sdk/agrifood/agrifood-farming-rest/samples/v1-beta/javascript/sample.env

```diff
@@ -0,0 +1,12 @@
+# Used to authenticate using Azure AD as a service principal for role-based authentication.
+#
+# See the documentation for `EnvironmentCredential` at the following link:
+# https://docs.microsoft.com/javascript/api/@azure/identity/environmentcredential
+AZURE_TENANT_ID=72f988bf-86f1-41af-91ab-2d7cd011db47
+AZURE_CLIENT_ID=39d176c5-91b6-454b-854d-55813c17d92d
+AZURE_CLIENT_SECRET=TOo8Q~_bZFdCLspdxJLrh-zRCCpYrf56-2D4FbJ5
+TEST_MODE=record
+# Used in most samples. Retrieve these values from a Azure FarmBeats instance
```

Comment:
Make sure to get rid of these files with secrets. Also make sure to reset your resource keys